src/main/resources/swagger/api-docs.yaml

```diff
@@ -524,6 +524,13 @@ paths:
           required: true
           type: string
           in: path
+        - name: workspace
+          description: Cloned workspace information
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: WorkspaceIngest
```

Comment:
This is matching Rawls functionality.  Rawls was updated to take the full ingest object and do an attribute update as part of the clone.  Not yet in the UI, but this would allow the user to update the description at clone time as well.  It will be the UI's responsibility to make sure we're not allowing nonsensical realms (or we'll get an error from Rawls).